### DISCUSSION TITLE
Question: build for chart.js 2.9.3

### DISCUSSION THREAD

**Discussion by g-ackolad** [kurkle/chartjs-chart-sankey#37] [Q&A] [answered]
Hi! 

I would reeeeeally love to use this library, as it does exactly what I need in a project. I am however confined to version 2.9.3 of Chart.js and the sankey chart does not work for that.

Would it be possible to build a version working for 2.9.3?

I've tried it myself but since I'm not very good at the architecture of Chart.js I can't seem to get it to work. It keeps complaining about Cannot read property xAxes of undefined.

Thank you for an awesome library!

**Comment by kurkle** [comment, marked as answer]
Hi, unfortunately it would not be a 5 minute job and its not in my interests.
Why are you confined to v2?

**Comment by g-ackolad** [comment]
We use Chart.js extensively in the project and would have to rewrite alot of code to make it work with the 3.0.0 architecture.
I fully understand :) Thank you for answering and I hope we can use this lib in the future!
